aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -299,6 +309,18 @@ class WriterJsonTar(ArchiveWriterAbstract):
     The entire containing folder is then compressed as a tar file.
     """
 
+    def __init__(self, filename: str, progress_context: Optional[ProgressContext] = None, **kwargs: Any):
+        """A writer for zipped archives.
+
+        :param filename: the filename (possibly including the absolute path)
+            of the file on which to export.
+        :param progress_context: A context manager for creating and updating a progress bar
+        :param sandbox_in_repo: Create the temporary uncompressed folder within the aiida repository
```

Comment:
should this be made an explicit argument then?

Reply:
agree, but the problem I had is that mypy then complains about the signature (same for `use_compression` in the zip writer). I'll see if there is a workaround, or just use `type: ignore`.

Reply:
done